crates/graphql/src/schema.rs

```diff
@@ -680,6 +684,75 @@ impl<'a> From<models::AuctionHouse<'a>> for AuctionHouse {
     }
 }
 
+#[derive(Debug, Clone)]
+struct PublicBid {
+    pub address: String,
+    pub trade_state: String,
+    pub wallet: String,
+    pub token_mint: String,
+    pub price: UnsignedInt8,
+    pub trade_state_bump: i32,
+    pub activated_at: Option<UnsignedInt8>,
+}
+
+#[juniper::graphql_object(Context = AppContext)]
+impl PublicBid {
+    pub fn address(&self) -> String {
+        self.address.clone()
+    }
+
+    pub fn trade_state(&self) -> String {
+        self.trade_state.clone()
+    }
+    pub fn wallet(&self) -> String {
+        self.wallet.clone()
+    }
+    pub fn token_mint(&self) -> String {
+        self.token_mint.clone()
+    }
+    pub fn price(&self) -> UnsignedInt8 {
+        self.price
+    }
+    pub fn trade_state_bump(&self) -> i32 {
+        self.trade_state_bump
+    }
+
+    pub fn activated_at(&self) -> Option<UnsignedInt8> {
```

Comment:
Can we convert these to datetime utc?

Same with closed_at

Reply:
May be best to do this conversion when saving to the db. use datatime col type in postgres.


Then we can add a custom scaler for DateTime if not supported natively by juniper.